CHANGELOG.md

```diff
@@ -1,3 +1,82 @@
+## Release 0.11.0
+
+New features:
+
+- UI Terminal to attach to Docker containers and see their output or `EXEC` a shell on them
```

Comment:
"Add a terminal to the UI with the ability to `attach` to, or `exec` a shell in, a Docker container."